packages/web/src/utils.test.ts

```diff
@@ -0,0 +1,62 @@
+import { parseProperties } from './utils';
+
+describe('utils', () => {
+  describe('parse properties', () => {
+    describe('strip', () => {
+      it('should allow all properties', () => {
+        const properties = {
+          number: 10,
+          string: 'some-string',
+          boolean: true,
+          nullable: null,
+        };
+
+        const parsed = parseProperties(properties, { strip: true });
+
+        expect(properties).toEqual(parsed);
+      });
+
+      it('should dismiss array and object', () => {
+        const properties = {
+          string: 'some-string',
+          array: [],
+          object: {},
+        };
+
+        const parsed = parseProperties(properties, { strip: true });
+
+        expect({
+          string: 'some-string',
+        }).toEqual(parsed);
+      });
+    });
+
+    describe('throw error', () => {
+      it('should allow all properties', () => {
+        const properties = {
+          number: 10,
+          string: 'some-string',
+          boolean: true,
+          nullable: null,
+        };
+        const parsed = parseProperties(properties, {
+          strip: false,
+        });
+
+        expect(properties).toEqual(parsed);
+      });
+
+      it('should throw an error for arrayProp and objectProp', () => {
+        const properties = {
+          string: 'some-string',
+          arrayProp: [],
+          objectProp: {},
+        };
+
+        expect(() => {
+          parseProperties(properties, { strip: false });
```

Comment:
Let's use a variable like in the other test above

Reply:
its a closure thats required for catching errors. we could move it out but I don't see a reason to think about about a name + we're not reusing it.

Reply:
Oh, true! Sorry, makes sense!